### ISSUE TITLE
How to evaluate algorithm performance

### ISSUE THREAD

**Issue by AliceInCh** [git-disl/TOG#10]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by khchow-gt** [comment]
Mean average precision (mAP) is the commonly used metric for performance evaluation. You can find the implementation here: https://github.com/chenyuntc/simple-faster-rcnn-pytorch/blob/367db367834efd8a2bc58ee0023b2b628a0e474d/utils/eval_tool.py#L11

**Comment by nikunjpansari** [comment]
I thought you would have integrated the detections, ground truth label, and coordinates of the BBox from the main attack code with the eval_tools.py which computes the metrics.  I was facing some issues in getting the metrics for different attacks. Could you help me integrate this eval_tools.py with attack code so that when I run the model network for the entire test image set, I get the Accuracy, AP, and mAP?

**Comment by wangxueping78** [comment]
> I thought you would have integrated the detections, ground truth label, and coordinates of the BBox from the main attack code with the eval_tools.py which computes the metrics. I was facing some issues in getting the metrics for different attacks. Could you help me integrate this eval_tools.py with attack code so that when I run the model network for the entire test image set, I get the Accuracy, AP, and mAP?

I also encountered this problem: I have trained the model now and want to get the mAP before and after the attack during the test. Have you solved this problem?Can you share it？
